pallets/did/src/benchmarking.rs

```diff
@@ -37,6 +39,7 @@ const AUTHENTICATION_KEY_ID: KeyTypeId = KeyTypeId(*b"0000");
 const ATTESTATION_KEY_ID: KeyTypeId = KeyTypeId(*b"0001");
 const DELEGATION_KEY_ID: KeyTypeId = KeyTypeId(*b"0002");
 const UNUSED_KEY_ID: KeyTypeId = KeyTypeId(*b"1111");
+const MAX_SIGNATURE_BYTE_LENGTH: u32 = 5 * 1024 * 1024;
```

Comment:
Maybe not part of this PR, but should we make this part of the pallet `Config` then? Or, alternatively, should this value be the maximum between `Ed25519`, `Sr25519`, and `Ecdsa` signature lenghts? That would be more realistic, as the longest signature is 64 bytes and here we are definitely overestimating.